test-cases/performance/perf-regression-nosqlbench-latency-mixed-50r-50w.yaml

```diff
@@ -0,0 +1,21 @@
+test_duration: 360
+
+stress_cmd_m: "nosqlbench cql-tabular2 rampup-cycles=1M main-cycles=50M write_ratio=5 read_ratio=5 threads=150 striderate=10 strides=800 --progress console:120s rf=3 partsize=5000 driver=cql-scylla -v"
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+instance_type_loader: 'c4.2xlarge'
+gce_instance_type_db: 'e2-highmem-8'
+gce_instance_type_loader: 'e2-standard-4'
```

Comment:
A better approach is to have this test yaml and split these parameters into "configurations".
One can be for AWS, one can be for GCE.
Later (see below) you can have one for GCE with pd-ssd.

Reply:
Later we can have all the configurations we want like they have in the blog table.

Reply:
Fixed (split the configs into AWS and GCE configs)